uint16 task_id
float64 deadline
float64 x_position
float64 y_position
float64 z_orientation
